api/src/methods/get_module.rs

```diff
@@ -0,0 +1,234 @@
+use {
+    crate::{json_utils::parse_params_3, jsonrpc::JsonRpcError},
+    umi_app::{ApplicationReader, Dependencies},
+    umi_shared::primitives::{Address, ToMoveAddress},
+};
+
+pub async fn execute<'reader>(
+    request: serde_json::Value,
+    app: &ApplicationReader<'reader, impl Dependencies<'reader>>,
+) -> Result<serde_json::Value, JsonRpcError> {
+    let (number, account, module): (_, Address, String) = parse_params_3(request)?;
```

Comment:
Bikeshedding: I think it is strange to have the block number come first. In Ethereum RPC functions it always comes last.